ipatests/test_integration/test_commands.py

```diff
@@ -1136,3 +1136,32 @@ def test_cacert_manage(self):
             raiseonerr=False
         )
         assert result.returncode == 0
+
+    def test_ipa_adtrust_install_with_locale_issue8066(self):
+        """
+        This test checks that ipa-adtrust-install command runs successfully
+        on a system with locale en_IN.UTF-8 without displaying error below
+        'IndexError: list index out of range'
+
+        This is a testcase for Pagure issue
+        https://pagure.io/freeipa/issue/8066
+        """
+        # Set locale to en_IN.UTF-8
+        tasks.kinit_admin(self.master)
+        self.master.run_command(
+            ["localectl", "set-locale", "LC_TIME=en_IN.UTF-8"]
```

Comment:
IIRC after localectl set-locale you need to reboot. The alternative is to change the LC_TIME env var for the root user by editing ~/.bash_profile on the master.

Reply:
LC_TIME env var for root user has been added in bashrc file to avoid reboot.